src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Github } from "Assets/Icons/Github";
+import { LinkedIn } from "Assets/Icons/LinkedIn";
+import { Twitter } from "Assets/Icons/Twitter";
+function Footer() {
+  return (
+    <footer className="text-gray-600 body-font">
+      <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+        <a className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+       
+          <img
+            src={require("./../../Assets/Images/FooterImage.png")}
```

Comment:
this is new to me that we can also import like that, great!

Reply:
Yeah we can do that mahendra